atef/widgets/happi.py

```diff
@@ -201,7 +201,17 @@ def _list_view_context_menu(self, pos: QtCore.QPoint) -> None:
         """Context menu for the happi list view."""
         self.menu = QtWidgets.QMenu(self)
         index: QtCore.QModelIndex = self.happi_list_view.indexAt(pos)
+
         if index is not None:
+            # Add action to add the selected device
+            add_action = self.menu.addAction(f'&Add Device: {index.data()}')
```

Comment:
I know you're just adding to this menu, perhaps it's worth fixing this bug while you're at it:

If I don't select a device, I still get the menu:
<img width="298" alt="image" src="https://user-images.githubusercontent.com/5139267/179304120-73ab6ae7-8128-4496-8d73-63d42240ebbf.png">